### ISSUE TITLE
Add User Info

### ISSUE THREAD

**Issue by jbickelhaupt** [washingtonstateuniversity/WSUWP-Platform#286]
When adding a new user to the site, it might benefit some clients to let them enter first and last names of each user they are granting access to. That way they can quickly filter by name and not just NID or email when scrolling through users.

![image](https://cloud.githubusercontent.com/assets/6163495/9947487/5fae1a8a-5d4f-11e5-9ca9-ae29e9a3ecb3.png)

**Comment by jeremyfelt** [comment]
Definitely. Possibly doing this via WordPress on https://core.trac.wordpress.org/ticket/18163, but we may want to do something earlier if that doesn't get traction.
